packages/console/src/pages/ConnectorDetails/index.tsx

```diff
@@ -47,7 +47,8 @@ const ConnectorDetails = () => {
 
   useEffect(() => {
     if (data) {
-      setConfig(JSON.stringify(data.config, null, 2));
+      const hasData = Object.keys(data.config).length > 0;
+      setConfig(hasData ? JSON.stringify(data.config, null, 2) : data.configTemplate);
```

Comment:
@xiaoyijun is this by design? i think redundant content will be filled

<img width="546" alt="image" src="https://user-images.githubusercontent.com/14722250/170508385-7f4f55b5-69c0-401a-961e-aa193ac30881.png">

Reply:
This aligns with other connectors and needs further discussion.

Reply:
cc: @wangsijie @darcyYe

Reply:
@demonzoo